### ISSUE TITLE
Can we zip state.json to stay under GitHubs strict file limit of 100MB

### ISSUE THREAD

**Issue by KasperZutterman** [TREEcg/LDES-Action#48]
Source: https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github

**Comment by pietercolpaert** [comment]
I would rephrase this issue as:

Problem: state.json is hitting max file size limit

Possible solutions:
 - gzipping state.json
 - splitting state.json in fragments
 - use a file-based store that is compressed by default. E.g., sqlite or leveldb
 - researching ways to keep less data, such as approximate membership functions?
